tests/bearlib/abstractions/LinterTest.py

```diff
@@ -154,7 +182,7 @@ def test_decorator_invalid_states(self):
 
     def test_decorator_generated_default_interface(self):
         uut = linter('some-executable')(self.ManualProcessingTestLinter)
-        with self.assertRaisesRegex(NotImplementedError, ''):
+        with self.assertRaises(NotImplementedError):
```

Comment:
Ah, better to either describe this in your commit message or make a separate issue/commit for this change :)